api/v1/users/PKPUserHandler.inc.php

```diff
@@ -318,54 +333,47 @@ private function _processAllowedparams($params, $allowedKeys)
 
         return $returnParams;
     }
+}
 
+class PKPUserHandler extends APIHandler
+{
     /**
-     * Retrieve the user report
-     *
-     * @param Slim\Http\Request $slimRequest Slim request object
-     * @param \APIResponse $response Response
-     *
-     * @return ?\APIResponse Response
+     * Constructor
      */
-    public function getReport(\Slim\Http\Request $slimRequest, \APIResponse $response, array $args): ?\APIResponse
+    public function __construct()
     {
-        $request = $this->getRequest();
+        $this->_handlerPath = 'users';
 
-        $context = $request->getContext();
-        if (!$context) {
-            return $response->withStatus(404)->withJsonError('api.404.resourceNotFound');
-        }
+        $roles = implode(',', [Role::ROLE_ID_SITE_ADMIN, Role::ROLE_ID_MANAGER, Role::ROLE_ID_SUB_EDITOR]);
```

Comment:
Is it possible to continue to use the role IDs symbolically rather than collapsing them into a string?

Reply:
Unfortunately no, because middleware params on Laravel accept only single string or CSVs.

As [mentioned here](https://laravel.com/docs/7.x/middleware#middleware-parameters):

>Middleware parameters may be specified when defining the route by separating the middleware name and parameters with a :. Multiple parameters should be delimited by commas: